crates/re_query_cache/src/cache.rs

```diff
@@ -669,6 +675,15 @@ impl std::fmt::Debug for CacheBucket {
 }
 
 impl CacheBucket {
+    // Check invariants in debug builds
+    fn sanity_check(&self) {
```

Comment:
Is rust able to optimize this whole thing out in release builds?

Something like:
```suggestion
    #[inline]
    fn sanity_check(&self) {
      if cfg!(debug_assertions) {
```
would probably avoid the question.

Reply:
agreed

Reply:
I'll just do it in the interest of time